framework/SwiftPlot/Axis.swift

```diff
@@ -0,0 +1,10 @@
+public class Axis{
+  public var scaleX: Float = 1
+	public var scaleY: Float = 1
+  public var plotMarkers: PlotMarkers = PlotMarkers()
+  public var series = [Series]()
+  public init(){}
+
+  public static let PRIMARY_AXIS: Int = 0
+  public static let SECONDARY_AXIS: Int = 1
+}
```

Comment:
In Swift, even static constants tend not to be all-uppercase like in C-family languages. I'd name these like you would a property (lowercase first letter, camelcase after that).

Reply:
As a broader design issue, I'd make the primary / secondary axis a true enum and not a hardcoded numeric value. You want to make these cases exhaustive, and starting to work with numbers for them opens you up to the issues inherent with numerical enums elsewhere.

This could be an enum type contained within Axis, such as AxisLocation (or just Location, because its namespace would be Axis.Location as a subtype) or outside of it if other types would use it. I'd do it inside Axis for now, and see how that works.